prow/github/types.go

```diff
@@ -214,14 +214,15 @@ const (
 
 // PullRequestChange contains information about what a PR changed.
 type PullRequestChange struct {
-	SHA       string `json:"sha"`
-	Filename  string `json:"filename"`
-	Status    string `json:"status"`
-	Additions int    `json:"additions"`
-	Deletions int    `json:"deletions"`
-	Changes   int    `json:"changes"`
-	Patch     string `json:"patch"`
-	BlobURL   string `json:"blob_url"`
+	SHA              string `json:"sha"`
+	Filename         string `json:"filename"`
+	Status           string `json:"status"`
+	Additions        int    `json:"additions"`
+	Deletions        int    `json:"deletions"`
+	Changes          int    `json:"changes"`
+	Patch            string `json:"patch"`
+	BlobURL          string `json:"blob_url"`
+	PreviousFilename string `json:"previous_filename"`
```

Comment:
Nice, this field is not only undocumented, but not even referenced in the example response payload in GitHub's docs: https://developer.github.com/v3/pulls/#list-pull-requests-files

(Are we sure it is real?)

Reply:
yeah we did an actual api call... their example does not list all valid status :-(

Reply:
lol, I got it in debugging https://github.com/kubernetes/test-infra/issues/8891, I think we can probably depend on it? most of github's API fields are not properly documented, FWIW.